### PR TITLE
[SDK-3695] Update README based on new design

### PR DESCRIPTION
**Pull request by jimmyjames**
Updates the README per the new design. 

Note that the banner is not yet finalized, so this PR does not include that.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,140 +58,62 @@ Decode it using any JWT library or tool like [jwt.io](https://jwt.io/?value=eyJ0
...
+---
 
-[![FOSSA Status](https://app.fossa.com/api/projects/git%2Bgithub.com%2Fauth0%2Fjwks-rsa-java.svg?type=large)](https://app.fossa.com/projects/git%2Bgithub.com%2Fauth0%2Fjwks-rsa-java?ref=badge_large)
\ No newline at end of file
+<p align="center">
+  <picture>
+    <source media="(prefers-color-scheme: light)" srcset="./assets/auth0_light_mode.png"   width="150">
+    <source media="(prefers-color-scheme: dark)" srcset="./assets/auth0_dark_mode.png" width="150">
+    <img alt="Auth0 Logo" src="./auth0_light_mode.png" width="150">
+  </picture>
```
Comment (stevehobbsdev) [inline README.md:115]: We now have these URLs in the CDN, see https://auth0.slack.com/archives/C0414QJGXG9/p1666171760950799?thread_ts=1666170245.671199&cid=C0414QJGXG9

**Review by stevehobbsdev — COMMENTED** [review]
LGTM 👍🏻

File: README.md (hunk 1 above)
Comment (jimmyjames) [reply README.md]: Updated to use the CDN and removed the local assets

File: README.md [hunk 2]
```diff
@@ -1,38 +1,52 @@
...
 
-[![CircleCI](https://circleci.com/gh/auth0/jwks-rsa-java.svg?style=svg)](https://circleci.com/gh/auth0/jwks-rsa-java)
-[![Maven Central](https://img.shields.io/maven-central/v/com.auth0/jwks-rsa.svg)](http://search.maven.org/#search%7Cga%7C1%7Cg%3A%20com.auth0%20a%3Ajwks-rsa)
-[![FOSSA Status](https://app.fossa.com/api/projects/git%2Bgithub.com%2Fauth0%2Fjwks-rsa-java.svg?type=shield)](https://app.fossa.com/projects/git%2Bgithub.com%2Fauth0%2Fjwks-rsa-java?ref=badge_shield)
+[![CircleCI](https://img.shields.io/circleci/project/github/auth0/jwks-rsa-java.svg?style=flat-square)](https://circleci.com/gh/auth0/jwks-rsa-java/tree/master)
+[![Coverage Status](https://codecov.io/gh/auth0/jwks-rsa-java/branch/master/graph/badge.svg?style=flat-square)](https://codecov.io/github/auth0/jwks-rsa-java)
+[![License](http://img.shields.io/:license-mit-blue.svg?style=flat)](https://doge.mit-license.org/)
+[![Maven Central](https://img.shields.io/maven-central/v/com.auth0/jwks-rsa-java.svg?style=flat-square)](https://search.maven.org/#search%7Cga%7C1%7Cg%3A%20com.auth0%20a%3Ajwks-rsa)
```
Comment (ewanharris) [inline README.md:6]: I think this should drop the `-java`
```suggestion
[![Maven Central](https://img.shields.io/maven-central/v/com.auth0/jwks-rsa.svg?style=flat-square)](https://search.maven.org/#search%7Cga%7C1%7Cg%3A%20com.auth0%20a%3Ajwks-rsa)
```

**Review by ewanharris — CHANGES_REQUESTED** [review]
Just the maven central version, but looks good other than that!

**Review by ewanharris — APPROVED** [review]

**Pull request merged**